src/solvers/dg/3d/dg.jl

```diff
@@ -2442,12 +2563,54 @@ function calc_blending_factors!(alpha, alpha_pre_smooth, u,
 
       alpha[large] = max(alpha_pre_smooth[large], 0.5 * alpha_pre_smooth[lower_left],  alpha[large])
       alpha[large] = max(alpha_pre_smooth[large], 0.5 * alpha_pre_smooth[lower_right], alpha[large])
-      alpha[large] = max(alpha_pre_smooth[large], 0.5 * alpha_pre_smooth[upper_left],  alpha[large])
       alpha[large] = max(alpha_pre_smooth[large], 0.5 * alpha_pre_smooth[upper_right], alpha[large])
+      alpha[large] = max(alpha_pre_smooth[large], 0.5 * alpha_pre_smooth[upper_left],  alpha[large])
```

Comment:
Why?

Reply:
No idea...maybe from the merge with master?

Reply:
Okay, might be the case. I would favor keeping the same order as everywhere else.

Reply:
changed back.